config/initializers/rgeo.rb

```diff
@@ -0,0 +1,6 @@
+# frozen_string_literal: true
+
+RGeo::ActiveRecord::SpatialFactoryStore.instance.tap do |config|
+  config.register(RGeo::Geographic.spherical_factory(srid: 4326, uses_lenient_assertions: true),
```

Comment:
I think a comment here explaining why this initializer—especially the `uses_lenient_assertions` part—is important would help us or someone else in the future.